client/lcd/swagger-ui/swagger.yaml

```diff
@@ -753,15 +753,25 @@ paths:
           description: Invalid delegator address
         500:
           description: Internal Server Error
-  /stake/delegators/{delegatorAddr}/redelegations:
+  /stake/redelegations:
```

Comment:
why would you want to get all the redelegations in the state ? there's no use case for that

Reply:
Maybe not in Voyager?  But that totally seems like something some block explorer might want.

Reply:
but you can still go through the list of validators and do that

Reply:
and through the `/txs` endpoint

Reply:
> and through the /txs endpoint

Either through that or with the `GET /stake/validators/{validatorAddr}/redelegations` endpoint